### ISSUE TITLE
Setup explicit GitHub Action to deploy eicug.github.io; manual deploy to eicug.org

### ISSUE THREAD

**Issue by wdconinc** [eicug/eicug.github.io#44]
With GitHub Actions we can deploy to the eicug.github.io location for staging, then have a manual job to deploy to eicug.org. That would put the entire functionality in the same interface instead of requiring an ssh session to deploy to production. It would make the deploy experience easier without leaking credentials (if @buddhasystem puts them in the settings as a repo secret).

Why? I apparently don't have the right production deploy secrets or I am doing something else wrong, and in any case most of the handling of PRs on this repo is an on-the-go kind of thing for me.

**Comment by buddhasystem** [comment]
I don't quite follow, sorry. Do you mean an automatic copy over to the eicug.org server, as a part of CI action?

**Comment by wdconinc** [comment]
Not automatic. A manual step in GitHub Actions.

**Comment by vskokov** [comment]
I agree, we should explore this more. Right now I just wrote a bash script on the server side, but it had to be run manually on the server,
